doc/reference/reference_lua/box_info/synchro.rst

```diff
@@ -94,51 +109,53 @@ box.info.synchro
     Create a space called ``sync`` and enable synchronous replication on this space.
     Then, create an index.
 
-    ..  code-block:: tarantoolsession
+    ..  code-block:: console
 
-        tarantool> s = box.schema.space.create("sync", {is_sync=true})
-        tarantool> _ = s:create_index('pk')
+        app:instance001> s = box.schema.space.create("sync", {is_sync=true})
+        app:instance001> _ = s:create_index('pk')
 
     After that, use ``box.ctl.promote()`` function to claim a queue:
 
-    ..  code-block:: tarantoolsession
+    ..  code-block:: console
 
-        tarantool> box.ctl.promote()
+        app:instance001> box.ctl.promote()
 
     Next, perform data manipulations:
 
-    ..  code-block:: tarantoolsession
+    ..  code-block:: console
 
-        tarantool> require('fiber').new(function() box.space.sync:replace{1} end)
+        app:instance001> require('fiber').new(function() box.space.sync:replace{1} end)
         ---
         - status: suspended
           name: lua
-          id: 119
+          id: 127
         ...
-        tarantool> require('fiber').new(function() box.space.sync:replace{1} end)
+        app:instance001> require('fiber').new(function() box.space.sync:replace{1} end)
         ---
         - status: suspended
           name: lua
-          id: 120
+          id: 128
         ...
-        tarantool> require('fiber').new(function() box.space.sync:replace{1} end)
+        app:instance001> require('fiber').new(function() box.space.sync:replace{1} end)
         ---
         - status: suspended
           name: lua
-          id: 121
+          id: 129
         ...
 
     If you call the ``box.info.synchro`` command again,
     you will see that now there are 3 transactions waiting in the queue:
 
-    ..  code-block:: tarantoolsession
+    ..  code-block:: console
 
-        tarantool> box.info.synchro
+        app:instance001> box.info.synchro
         ---
         - queue:
             owner: 1
-            term: 2
-            len: 3
+            confirm_lag: 0
+            term: 29
+            age: 0
+            len: 0
```

Comment:
Something is wrong in this example. `len` should be `3`.

 Also it's strange that  term was `28` and became `29`. There shouldn't be a term increase between the two `box.info.synchro` calls. Previously it started at `2` and remained `2`.